src/main/java/com/verily/genomewarp/utils/GenomeWarpTestUtils.java

```diff
@@ -154,6 +154,22 @@ public static boolean equivalentRanges(List<GenomeRange> listOne,
     return true;
   }
 
+  public static boolean equivalentGenomeRanges(Map<String, List<GenomeRange>> first,
```

Comment:
Please you add a comment explaining the logic of what makes ranges equivalent.

Reply:
Done.